security/cmd/node_agent/main.go

```diff
@@ -59,7 +59,7 @@ func init() {
 	cAClientConfig.CAAddress = env.RegisterStringVar(envvar.CAAddress, "", "CA's endpoint").Get()
 	cAClientConfig.CAProviderName = env.RegisterStringVar(envvar.CAProvider, "", "CA's provider name").Get()
 	cAClientConfig.CAProtocol = env.RegisterStringVar(envvar.CAProtocol, nvm.IstioCAService, "CA service protocol").Get()
-	cAClientConfig.Env = env.RegisterStringVar(envvar.NodeEnv, "unspecified", "Node Environment : unspecified | onPremVM | gcpVM | awsVM").Get()
+	cAClientConfig.Env = env.RegisterStringVar(envvar.NodeEnv, "unspecified", "Node Environment : Unspecified | OnPremVM | GcpVM | AwsVM").Get()
```

Comment:
why options are capitalized for the first letter?

Reply:
To match with the variable names (for outside package use). https://github.com/istio/istio/blob/701ca367f2d4b04cf273f0b6fdb5485109aa9ee0/security/pkg/platform/client.go#L43